homeassistant/components/wiz/manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "domain": "wiz",
+  "name": "WiZ Platform",
```

From their website looks like it's just WiZ
```suggestion
  "name": "WiZ",
```